code/src/reducers/tasks.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const tasks = createSlice({
+  name: 'tasks',
+  initialState: {
+    todos: [
+      { id: 1, title: 'groceries', isDone: false },
+      { id: 2, title: 'pick up kids from day care', isDone: false },
+      { id: 3, title: 'traveling on Friday', isDone: false }
+    ]
+  },
```

Comment:
If you don't want to start with an an empty list, here his how i did: 

`const initialState = {
  items: []
};

const todos = createSlice({
  name: 'todos',
  initialState,
  reducers: {
    addToDo: (store, action) => {
      const newToDo = {
        id: uniqid(),
        text: action.payload,
        isComplete: false,
        postedTime: moment().format('MMM Do YY')
      };
      store.items = [newToDo, ...store.items]
    },`